L1Trigger/Configuration/python/customiseSettings.py

```diff
@@ -2,6 +2,86 @@
 import os.path
 import FWCore.ParameterSet.Config as cms
 
+def L1TSettingsToCaloParamsHI_2023_v0_4(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParamsHI_2023_v0_4_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2023_v0_4_ppRef(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2023_v0_4_ppRef_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2023_v0_3_ppRef(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2023_v0_3_ppRef_cfi")
+    return process
+
+def L1TSettingsToCaloParamsHI_2023_v0_3(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParamsHI_2023_v0_3_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2023_v0_3(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2023_v0_3_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2023_v0_2(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2023_v0_2_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2023_v0_1(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2023_v0_1_cfi")
+    return process
+
+def L1TSettingsToCaloParamsHI_2023_v0_0(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParamsHI_2023_v0_0_cfi")
+    return process
+
+def L1TSettingsToCaloParamsHI_2022_v0_6(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParamsHI_2022_v0_6_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2022_v0_6(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2022_v0_6_cfi")
+    return process
+
+def L1TSettingsToCaloParamsHI_2022_v0_5(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParamsHI_2022_v0_5_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2022_v0_5(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2022_v0_5_cfi")
+    return process
+
+def L1TSettingsToCaloParamsHI_2022_v0_4_1(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParamsHI_2022_v0_4_1_cfi")
+    return process
+
+def L1TSettingsToCaloParamsHI_2022_v0_4(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParamsHI_2022_v0_4_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2022_v0_4(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2022_v0_4_cfi")
+    return process
+
+def L1TSettingsToCaloParamsHI_2022_v0_3(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParamsHI_2022_v0_3_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2022_v0_3(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2022_v0_3_cfi")
+    return process
+
+def L1TSettingsToCaloParamsHI_2022_v0_2(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParamsHI_2022_v0_2_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2022_v0_2(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2022_v0_2_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2022_v0_1(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2022_v0_1_cfi")
+    return process
+
```

Comment:
I think this will have the same problem as previously, most of these don't seem to exist currently in the main CMSW

Reply:
@bundocka Should these files be added in main CMSSW?

Reply:
yes they should. likely along with some other things from the integration branch. this isn't urgent though, since these customisations aren't used in any central CMSSW workflow.

Reply:
> Fine by me. Just bear in mind someone else may ask you to change it.

Me, for example :-)

Please either remove the definitions here, or add the missing configs: in both cases it will be fast